jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,891 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+Nim version 1.6 is now officially released!
+
+A year in the making, 1.6 is the latest stable release and by far the largest yet.
+We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in more than 40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and in the last 2 months the top-50 ([https://forum.nim-lang.org/t/8297](https://forum.nim-lang.org/t/8297)).
+We hope this release will reinforce this trend, building on Nim's core strengths:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; Lisp-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
+
+Why use Nim?
+* one language to rule them all: write your iOS/Android/web frontend and backend in the same language, from a [Gameboy kickstarter](https://forum.nim-lang.org/t/8375) to a [blockchain client](https://github.com/status-im).
+* concise and readable like Python: `echo "hello world"` is a 1-liner (5 lines in C, C++, Go, Java).
+* small binaries: `echo "hello world"` generates a 73K binary (Go: 2MB, Rust: 377K, C++: 56K) [1].
+* fast compile times: a full compiler rebuild takes ~12s (Rust: 15min, gcc: 30min+, clang: 1hr+, Go: 90s) [2].
+* C-like performance: see [Web Frameworks Benchmark](https://web-frameworks-benchmark.netlify.app/result), [ray tracing](https://nim-lang.org/blog/2020/06/30/ray-tracing-in-nim.html), [primes](https://github.com/PlummersSoftwareLLC/Primes)
+* no need for makefiles, cmake, configure or other build scripts, thanks to CTFE and dependency tracking [3]
+* target any platform with a C compiler: [embedded systems](https://github.com/elcritch/nesper), [micro-controllers](https://forum.nim-lang.org/t/7731), [WASM](https://forum.nim-lang.org/t/4779), Nintendo Switch or [Gameboy](https://www.kickstarter.com/projects/penguinrik/goodboy-galaxy-exploration-platform-game-gba-pc-and-switch)
```

Comment:
This should be "Game Boy Advance", and can link to the forum post (https://forum.nim-lang.org/t/8375).

Note that the original Game Boy is way less C-friendly than GBA ^^

Reply:
thanks! done

Reply:
Not sure if we should link the NLVM thread for wasm, as nlvm is quite outdated (Nim 1.2) compared to using Emscripten

Reply:
Let's link nico (web game based on WASM) instead or treeform's tutorial instead!